runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -464,17 +469,32 @@ public void onAnimationChanged(AnimationChanged event)
 	{
 		Actor actor = event.getActor();
 
-		if (actor != client.getLocalPlayer())
-		{
-			return;
-		}
-
 		if (config.showVengeanceOther()
+			&& actor == client.getLocalPlayer()
 			&& actor.getAnimation() == AnimationID.ENERGY_TRANSFER_VENGEANCE_OTHER
 			&& actor.getInteracting().getGraphic() == VENGEANCEOTHER.getGraphicId())
 		{
 			createGameTimer(VENGEANCEOTHER);
 		}
+
+		String actorName = actor.getName();
+
+		if (config.showAbyssalSireStun()
+			&& actor instanceof NPC
+			&& actorName != null
+			&& actorName.equals("Abyssal Sire"))
```

Comment:
Remove the `actorName` assignment and null check, and change this check to `"Abyssal Sire".equals(actor.getName())`